index.html

```diff
@@ -38,7 +43,8 @@ <h2>Glad to see you!</h2>
   
 <section class="works"> 
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 Most of the elements in your `works` section have an error in their positioning, so when you change the `width` value to the desired values, they are not positioned responsively and create unnecessary spaces. Please correct them🙏🙏🙏 

- ### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/199764491-42d8771e-e6aa-4b84-aeab-f6714f7b2e08.png)

- ### Figma design ✔️ 

![image](https://user-images.githubusercontent.com/63915024/199764618-33d43a1b-d7f1-48ce-aa64-b277ec012fd0.png)

Reply:
Fixed.